cfgov/paying_for_college/jinja2/paying-for-college/disclosure_feedback.html

```diff
@@ -0,0 +1,61 @@
+{% extends 'layout-full.html' %}
+
+{% block title %}
+    Paying for College - Compare financial aid and college cost | Consumer Financial Protection Bureau
+{% endblock %}
+
+{% block content_main %}
+
+<header class="block block__flush-top">
+    <h1>Paying for College</h1>
+    <p class="lead-paragraph">Understanding your financial aid offer</p>
```

Comment:
Similarly, the H1 should be more specific, and this lead paragraph becomes superfluous at that point,

```suggestion
    <h1>Give us feedback on Paying for College</h1>
```